### PR TITLE
kie-issues#37: Exclude 'WEB-INF' folder and html files of gwt-editors from 'dist' folder

### PR DESCRIPTION
**Pull request by jomarko**
This PR closes:
- https://github.com/kiegroup/kie-issues/issues/37
- https://issues.redhat.com/browse/KOGITO-1430

**dist** folder before changes
Screenshot from 2022-11-25 08-10-05

**dist** folder after changes:
Screenshot from 2022-11-25 08-06-21

### CODE REVIEW

**Comment by jomarko** [comment]
All CI checks passed @yesamer @tiagobento

**Review by tiagobento — APPROVED** [review]

**Review by yesamer — APPROVED** [review]

File: packages/chrome-extension-serverless-workflow-editor/webpack.config.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ module.exports = async (env) => {
           {
             from: swEditor.swEditorPath(),
             to: "./diagram",
-            globOptions: { ignore: ["**/WEB-INF/**/*"] },
+            globOptions: { ignore: ["**/WEB-INF/**/*", "**/*.html"] },
```
Comment (caponetto) [inline packages/chrome-extension-serverless-workflow-editor/webpack.config.js:81]: `chrome-extension-pack-kogito-kie-editors` package also needs this change, doesn't it?

File: packages/dashbuilder-viewer-deployment-webapp/webpack.config.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ module.exports = async (env) =>
           {
             from: dashbuilderClient.dashbuilderPath(),
             to: "./",
-            globOptions: { ignore: ["WEB-INF/**/*"] },
+            globOptions: { ignore: ["**/WEB-INF/**/*", "**/*.html"] },
```
Comment (caponetto) [inline packages/dashbuilder-viewer-deployment-webapp/webpack.config.js]: Please revert this change. Dashbuilder uses some `index.html` files to load external components.

File: packages/vscode-extension-dashbuilder-editor/webpack.config.js [hunk 3]
```diff
@@ -62,12 +62,12 @@ module.exports = async (env) => [
...
             from: "./src/setup.js",
             to: "webview/",
-            globOptions: { ignore: ["WEB-INF/**/*"] },
+            globOptions: { ignore: ["**/WEB-INF/**/*", "**/*.html"] },
           },
           {
             from: dashbuilderClient.dashbuilderPath(),
             to: "webview/",
-            globOptions: { ignore: ["WEB-INF/**/*"] },
+            globOptions: { ignore: ["**/WEB-INF/**/*", "**/*.html"] },
```
Comment (caponetto) [inline packages/vscode-extension-dashbuilder-editor/webpack.config.js]: Looks like the VS Code extension for Dashbuilder is not working properly. Can you please check?

**Review by caponetto — CHANGES_REQUESTED** [review]

File: packages/dashbuilder-viewer-deployment-webapp/webpack.config.js (hunk 2 above)
Comment (jomarko) [reply packages/dashbuilder-viewer-deployment-webapp/webpack.config.js]: done as 37a389b069a

File: packages/chrome-extension-serverless-workflow-editor/webpack.config.js (hunk 1 above)
Comment (jomarko) [reply packages/chrome-extension-serverless-workflow-editor/webpack.config.js:81]: done as 1620df8355b

File: packages/vscode-extension-dashbuilder-editor/webpack.config.js (hunk 3 above)
Comment (jomarko) [reply packages/vscode-extension-dashbuilder-editor/webpack.config.js]: @caponetto sure. this extension is new for me. Would you mind to help me in understanding how can I built it locally? It is ntot described in https://github.com/kiegroup/kie-tools/blob/main/packages/vscode-extension-dashbuilder-editor/README.md

Or is the only way I need to try it to download CI build artifacts?

Comment (caponetto) [reply packages/vscode-extension-dashbuilder-editor/webpack.config.js]: To build and run this extension locally you can follow the same steps as the other VS Code extensions.
Then, open any `*.dash.yaml` file (you can get one example at https://start.kubesmarts.org).

Comment (jomarko) [reply packages/vscode-extension-dashbuilder-editor/webpack.config.js]: I downloaded build form CI, installed extension and opened example https://raw.githubusercontent.com/kiegroup/kie-tools/main/examples/dashbuilder/kitchensink/Kitchensink.dash.yml

This is what I got

what should I focus on there please?
Screenshot from 2022-11-30 10-52-08

Comment (caponetto) [reply packages/vscode-extension-dashbuilder-editor/webpack.config.js]: Do the "External Components" tab and its inner tabs load ok?
I installed the extension from CI yesterday and I was seeing a blank screen. Let me try again.

**Review by caponetto — APPROVED** [review]

**Comment by jomarko** [comment]
@tiagobento may I ask you for a merge?
